launch/launch_utils/launch_base.py

```diff
@@ -1,11 +1,19 @@
 # Don't launch this file directly, rather use the launch files one level up instead
 import os
 from launch_ros.actions import Node
+import subprocess
+from pathlib import Path
 
 def get_base_nodes():
     atos_dir = os.path.join(os.path.expanduser('~'), '.astazero', 'ATOS')
     params = os.path.join(atos_dir, 'conf', 'params.yaml')
 
+    #webgui logging
+    log = open(atos_dir / Path("webgui.log"), 'w')
+    # start webgui server
+    path = Path(__file__).parent.parent.parent.parent.parent.absolute() / Path("etc/simplecontrol/")
```

Comment:
Simpler to just enter the absolute path right away. Also less error prone in case someone does not have their workspace in ~.

Will this access to /etc need root privileges?